### DISCUSSION TITLE
Startgen page not found

### DISCUSSION THREAD

**Discussion by ZioZenigata** [serenity-is/Serenity#6065] [General]
Hi, 
I'm a pro user. This link https://github.com/volkanceylan/StartSharp/releases/ is wrong. I need to download startgen. Coul'd you help me please? Thanks

**Comment by VictorTomaili** [comment]
If you are a pro user, you can contact us through support or sales email.

**Reply by ZioZenigata** [reply]
Ok sorry, I'll do it! Thanks for your reply
